### PR TITLE
Fis modularization

### PR DESCRIPTION
**Pull request by ChaoqunCHEN**
Add documentation to public APIs.

### CODE REVIEW

File: packages-exp/installations-exp/src/api/delete-installations.ts [hunk 1]
```diff
@@ -23,6 +23,8 @@ import { FirebaseInstallationsImpl } from '../interfaces/installation-impl';
 import { FirebaseInstallations } from '@firebase/installations-types-exp';
 
 /**
+ * Deletes the Firebase Installation and all associated data.
+ *
  * @public
```
Comment (andirayo) [inline packages-exp/installations-exp/src/api/delete-installations.ts:28]: what does this line mean?

File: packages-exp/installations-exp/src/api/get-installations.ts [hunk 2]
```diff
@@ -20,6 +20,8 @@ import { FirebaseInstallations } from '@firebase/installations-types-exp';
 import { _getProvider } from '@firebase/app-exp';
 
 /**
+ * Returns a Firebase Installation instance for the given app.
```
Comment (andirayo) [inline packages-exp/installations-exp/src/api/get-installations.ts:23]: Returns an instance of FirebaseInstallations associated with the given FirebaseApp instance.

File: packages-exp/installations-exp/src/api/get-token.ts [hunk 3]
```diff
@@ -24,6 +24,8 @@ import {
 import { FirebaseInstallations } from '@firebase/installations-types-exp';
 
 /**
+ * Returns an authentication token for the current Firebase Installation.
```
Comment (andirayo) [inline packages-exp/installations-exp/src/api/get-token.ts:27]: ... Installation auth token, identifying ...

File: packages-exp/installations-exp/src/api/on-id-change.ts [hunk 4]
```diff
@@ -20,18 +20,22 @@ import { FirebaseInstallationsImpl } from '../interfaces/installation-impl';
 import { FirebaseInstallations } from '@firebase/installations-types-exp';
 
 /**
+ * An user defined callback function that takes action when Installations ID changes.
```
Comment (andirayo) [inline packages-exp/installations-exp/src/api/on-id-change.ts:23]: [...] that gets triggered [...]
or
[...] that gets called [...]

File: packages-exp/installations-exp/src/api/on-id-change.ts [hunk 5]
```diff
@@ -20,18 +20,22 @@ import { FirebaseInstallationsImpl } from '../interfaces/installation-impl';
...
+ *
  * @public
  */
 export type IdChangeCallbackFn = (installationId: string) => void;
 /**
+ * An unsubscribe function that will remove the onIdChange callback when called.
```
Comment (andirayo) [inline packages-exp/installations-exp/src/api/on-id-change.ts:29]: Unsubscribe a callback function previously added via {@link #IdChangeCallbackFn}.

File: packages-exp/installations-types-exp/index.d.ts [hunk 6]
```diff
@@ -16,13 +16,15 @@
  */
 
 /**
+ * A Firebase Installation instance which is a required argument for all Firebase Installations operations.
```
Comment (andirayo) [inline packages-exp/installations-types-exp/index.d.ts:19]: Public interface of the FirebaseInstallations SDK.

**Review by andirayo — APPROVED** [review]

File: packages-exp/installations-exp/src/api/delete-installations.ts (hunk 1 above)
Comment (ChaoqunCHEN) [reply packages-exp/installations-exp/src/api/delete-installations.ts:28]: The `@public` is the annotation that used by the tool `api-extractor` for auto-generating public docs.

**Review by hsubox76 — APPROVED** [review]
Approved pending @egilmorez review of doc comment text.

**Pull request merged**